TicketBooking.Data/Infrastructure/UnitOfWork.cs

```diff
@@ -7,7 +8,13 @@ public class UnitOfWork : IUnitOfWork
         private readonly TicketBookingDbContext _context;
 
         public UnitOfWork(TicketBookingDbContext context)
-        { _context = context; }
+        {
+            _context = context;
+
+            Aircrafts = new AircraftRepository(_context);
```

Comment:
what happen ì we have many repositories?

Reply:
I don't get this point. Do you mean in case there are more than 1 or even a lot of repositories, is there any faster way to create instances of all of them?